src/main/webapp/app/localci/build-queue/build-queue.component.html

```diff
@@ -879,3 +879,38 @@ <h5 class="my-0">
         </button>
     </div>
 </ng-template>
+
+<ng-template #buildLogsModal let-modal>
+    <div class="modal-header">
+        <h5 class="modal-title">
+            <span jhiTranslate="artemisApp.buildQueue.logs.title"></span>
+            <span> {{ this.displayedBuildJobId }}</span>
+        </h5>
+        <button type="button" class="btn-close" aria-label="Close" (click)="modal.dismiss()"></button>
+    </div>
+    <div class="modal-body">
+        <table class="table table-borderless">
+            <tbody>
+                @for (logEntry of rawBuildLogs; track logEntry) {
+                    <tr class="build-output__entry">
+                        <td class="build-output__entry-date">{{ logEntry.time | artemisDate: 'long' : true : undefined : false : true }}</td>
+                        <td class="build-output__entry-text">
+                            @for (line of logEntry.logLines; track line) {
+                                <span>{{ line }}</span>
+                                <br />
+                            }
+                        </td>
+                    </tr>
+                }
```

Comment:
Can it happen that we do not receive any build logs or get an empty array? 

If yes I think we should add an empty block